src/app/custom/enrichers/enrichers-catalog.json

```diff
@@ -191,92 +208,109 @@
+        "objectifTDM": "https://services.istex.fr/2022/02/09/enrichissement-par-doi/",
+        "type": "metadata"
+    },
+        {
+        "id": "post-v1-tools",
+        "url": "https://terms-extraction.services.istex.fr/v1/tools/normalize",
+        "doc": "https://github.com/Inist-CNRS/web-services/tree/main/services/terms-extraction",
+        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=terms-extraction%20-%20Extraction%20de%20termes#/terms-extraction/post-v1-tools-normalize",
+        "objectifTDM": "https://services.istex.fr/2022/01/12/normaliser-un-texte-ou-un-terme/",
+        "type": "metadata"
     },
     {
-        "id": "post-v1-inspire-category-meta-category-json",
-        "url": "https://mapping-tools.services.istex.fr/v1/inspire-category/meta-category/json",
-        "doc": "https://gitbucket.inist.fr/tdm/web-services/tree/master/mapping-tools#v1%2finspire-category%2fmeta-category%2fjson",
-        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=mapping-tools%20-%20Utilisation%20de%20tables%20de%20correspondance#/mapping/post-v1-inspire-category-meta-category-json",
-        "objectifTDM": "https://services.istex.fr/2023/02/06/regroupement-des-categories-inspire-en-meta-categorie-in2p3/",
+        "id": "post-v1-detect-lang",
+        "url": "https://nlp-tools2.services.istex.fr/v1/detect-lang",
+        "doc": "https://gitbucket.inist.fr/tdm/web-services/tree/master/nlp-tools2#v1%2fdetect-lang",
+        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=nlp-tools2%20-%20Outils%20de%20NLP#/NLP/post-v1-detect-lang",
+        "objectifTDM": "https://services.istex.fr/2021/12/16/detection-de-la-llangue-dun-texte/",
         "type": "other"
     },
-    {
-        "id": "post-v1-first-name-gender",
-        "url": "https://authors-tools.services.istex.fr/v1/first-name/gender",
-        "doc": "https://gitbucket.inist.fr/tdm/web-services/tree/master/authors-tools#v1%2ffirst-name%2fgender",
-        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=authors-tools%20-%20Les%20web%20services%20qui%20se%20lancent%20sur%20le%20champs%20%27authors%27#/default/post_v1_first_name_gender",
-        "objectifTDM": "https://services.istex.fr/2023/04/26/detection-de-genre/",
-        "type": "classification"
+                      {
+        "id": "post-v1-textExtract",
+        "url": "https://pdf-text.services.istex.fr/v1/pdf2txt",
+        "doc": "https://github.com/Inist-CNRS/web-services/tree/main/services/pdf-text",
+        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=pdf-text%20-%20Transforme%20un%20PDF%20en%20texte%20%C3%A0%20partir%20de%20l%27URL#/pdf-text/post-v1-pdf2txt",
+        "objectifTDM": "https://services.istex.fr/extraction-du-texte-epure-de-pdf/",
+        "type": "prétraitement"
```

Comment:
<!-- TODO's are small, trivial, but necessary changes. Distinguishing todo comments from issues: or suggestions: helps direct the reader's attention to comments requiring more involvement.  -->
**todo:** ‏Use the key from `translations.tsv`: `"pretraitement"`
```suggestion
        "type": "pretraitement"
```